its/src/test/java/com/sonar/it/scanner/msbuild/utils/TestUtils.java

```diff
@@ -220,6 +221,10 @@ public static void runMSBuildWithBuildWrapper(Orchestrator orch, Path projectDir
     assertThat(r).isZero();
   }
 
+  public static void updateSetting(Orchestrator orchestrator, String projectKey, String propertyKey, List<String> values) {
+    newAdminWsClient(orchestrator).settings().set(new SetRequest().setComponent(projectKey).setKey(propertyKey).setValues(values));
```

Comment:
`setValue` returns 400: 
> Parameter 'value' must be used for single value setting. Parameter 'values' must be used for multi-value setting.

`setValues`, on the other hand, works as expected.
The behavior can be reproduced with Postman, but I cannot find a way to make setValue work.